config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java

```diff
@@ -60,8 +58,8 @@
     @Override public boolean isFirstTimeDeployment() { return isFirstTimeDeployment; }
     @Override public boolean useAdaptiveDispatch() { return useAdaptiveDispatch; }
     @Override public boolean useDedicatedNodeForLogserver() { return useDedicatedNodeForLogserver; }
-    @Override public boolean useFdispatchByDefault() { return useFdispatchByDefault; }
-    @Override public boolean dispatchWithProtobuf() { return dispatchWithProtobuf; }
+    @Override public boolean useFdispatchByDefault() { return false; }
```

Comment:
Remove implementations of `useFdispatchByDefault`/`dispatchWithProtobuf`.

Reply:
Done